src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X500NameEncoder.cs

```diff
@@ -542,6 +542,25 @@ private static byte[] ParseRdn(ReadOnlySpan<char> tagOid, ReadOnlySpan<char> cha
             return writer.Encode();
         }
 
+        private static bool IsForceUtf8Eligible(ReadOnlySpan<char> oid)
+        {
+            switch (oid)
+            {
+                case Oids.CommonName:
+                case Oids.GivenName:
+                case Oids.LocalityName:
+                case Oids.Organization:
```

Comment:
You added Initials in the known OIDs class, but didn't add it in the switch here.